content/en/cloud-computing.md

```diff
@@ -8,7 +8,24 @@ tags: ["infrastructure", "fundamental", ""]
 ## What it is
 
 Cloud computing offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
-We generally differentiate between private and public cloud, depending on whether the cloud infrastructure is exclusively dedicated to an organization or shared for open public services.
+It extends beyond merely providing resources and we generally differentiate between private and public cloud. 
+These models differ based on the usage and accessibility of the cloud infrastructure.
+
+In a private cloud, the infrastructure is exclusively dedicated to a single organization, 
+and it can exist both on-premises (within the organization's physical location) or off-premises (hosted by a third-party service provider). 
+This setup offers more control over the infrastructure, data security, and regulatory compliance, 
+and can be customized to meet specific business needs.
+
+On the other hand, a public cloud is a shared infrastructure available for open public services, 
+where resources are owned and operated by third-party cloud service providers. 
```

Comment:
While it's true that the infrastructure is shared, it might be misunderstood that the data is shared as well. It's important to clarify that while resources are shared in a public cloud, each user's data and applications remain isolated from others.

Reply:
Indeed ... Lets update that